### PR TITLE
Date fix

### LINKED ISSUE #123: Incorrect date displayed
Incorrect is being converted from API data for clock.

### PR DESCRIPTION
**Pull request by Paganiniana**
## Description

This PR replaces the _moment_ library with the original _countdown_ library used in the Vue-widget repository. There was a disparity in how they calculated dates, which has now been remedied.

Fixes #123

### CODE REVIEW

File: client/src/utils/countdown.ts [hunk 1]
```diff
@@ -0,0 +1,1434 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```
Comment (daniel-moon32) [inline client/src/utils/countdown.ts:2]: Is there a way we can import this using yarn instead of adding the full file?

**Review by daniel-moon32 — APPROVED** [review]
One comment about the utility file, but everything else looks good!

File: client/src/utils/countdown.ts (hunk 1 above)
Comment (Paganiniana) [reply client/src/utils/countdown.ts:2]: I tried using the proper NPM repo, found here https://github.com/mckamey/countdownjs, but it hasn't been updated in a little while. (That's fine for the Date computation part, but not fine for the module loading part). I spent a couple of hours trying to get Webpack to play nice with it, and finally gave up, using this solution ... but I'm not the savviest when it comes to Webpack, so if you have any alternatives, I'm all ears!

Comment (daniel-moon32) [reply client/src/utils/countdown.ts:2]: Probably not worth it to try to get it to work nicely. I think it works for now, but I think it's good to document that this is the case for maintenance.

**Pull request closed without merging**